pkg/api/latest/latest.go

```diff
@@ -21,7 +21,20 @@ var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
 // may be assumed to be most preferred to least preferred, and clients may
 // choose to prefer the earlier items in the list over the latter items when presented
 // with a set of versions to choose.
-var Versions = []unversioned.GroupVersion{{Group: "", Version: "v1"}}
+var Versions = []unversioned.GroupVersion{
```

Comment:
@deads2k is there any less-uglier way to determine if a resource is "openshift"?

(or this will go away once we have client-go that includes k8s client as well?).